app/models/concept/authorship.rb

```diff
@@ -0,0 +1,12 @@
+class Concept::Authorship < ApplicationRecord
+  belongs_to :concept,
+    inverse_of: :authorships,
+    foreign_key: :track_concept_id # TODO: Remove at ETL
+  belongs_to :author,
+    class_name: "User",
+    foreign_key: :user_id,
+    inverse_of: :authorships
+
+  # TODO: Remove at ETL
+  self.table_name = "concept_authorships"
```

Comment:
I think just name these to be correct now in the migrations as they're new tables